bot.py

```diff
@@ -720,9 +753,15 @@ async def addrepo(ctx, link):
 
 
 @bot.command(name="whois")
-async def whois(ctx, *, search):
+async def whois(
+    ctx,
+    *,
+    search: str = commands.parameter(
+        description="Mention, id, username, or display name to be searched"
+    ),
+):
     """
-    Looks up a person by username/display name in the verification sheet.
+    Looks up a person by username/display name in the database.
```

Comment:
The database is just an implementation detail. The original source of information that most users know about is the verification sheet.